hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -74,6 +81,14 @@ class AttributeSearchIntegrationTest : AbstractSearchIntegrationTest() {
           BodyPartBuilder(bodyPart = "Arm", comment = "love heart"),
           BodyPartBuilder(bodyPart = "Shoulder", comment = "dragon"),
         ),
+        scar = listOf(
+          BodyPartBuilder(bodyPart = "Arm", comment = "wrist"),
+          BodyPartBuilder(bodyPart = "Shoulder", comment = "bite"),
+        ),
+        mark = listOf(
+          BodyPartBuilder(bodyPart = "Arm"),
+          BodyPartBuilder(bodyPart = "Shoulder", comment = "birthmark"),
+        ),
```

Comment:
The key to testing nested queries is that this prisoner shouldn't match e.g. tattooo Arm/dragon (and similar for scar Arm/bite and mark Arm/birthmark).